src/DeviceInterface/AppleDeviceInterface.js

```diff
@@ -126,16 +131,31 @@ class AppleDeviceInterface extends InterfacePrototype {
      */
     async _show (details) {
         await this._showAutofillParent(details)
-        this._listenForSelectedCredential()
-            .then((response) => {
-                if (!response) {
-                    return
-                }
+        this._listenForSelectedCredential(async (response) => {
```

Comment:
🪄  We're using a callback here now instead of a promise so that we can call it multiple times depending on the response, rather than just one resolve with a promise